vars/publishHelmChartArtifactory.groovy

```diff
@@ -22,21 +22,7 @@ def call(params) {
     sh "helm repo add ${it.name} ${it.url}"
   }
   sh "helm package --dependency-update --version ${VERSION} --app-version ${VERSION} ${chartPath}/${chartName}"
-  rtServer (
-     id: "liatrio-artifactory",
-     url: params.get("helmRepository", "https://artifactory.liatr.io/artifactory/"),
-     credentialsId: params.get("helmRepositoryCredentials", "openshift-token")
-  )
-  rtUpload (
-    serverId: "liatrio-artifactory",
-    spec:
-      """{
-        "files": [
-          {
-            "pattern": "${chartName}-${VERSION}.tgz",
-            "target": "helm/"
-          }
-        ]
-     }"""
-  )
+  withCredentials([usernamePassword(credentialsId: 'jenkins-credential-artifactory', passwordVariable: 'artifactoryPassword', usernameVariable: 'artifactoryUsername')]) {
+    sh "curl -X PUT -u ${env.artifactoryUsername}:${env.artifactoryPassword} -T ${chartName}-${VERSION}.tgz 'https://artifactory.liatr.io/artifactory/helm/${chartName}-${VERSION}.tgz'"
```

Comment:
I would keep the ability to specify the repo in a parameter, rather than hardcoding

Reply:
Would be nice if the version could be a parameter instead of an environment variable